onyo/commands/edit.py

```diff
@@ -129,15 +135,18 @@ def edit(args, opdir: str) -> None:
             If user wants to discard changes, restore the asset's state
             """
             repo._git(['restore', str(asset)])
-            print(f"'{asset}' not updated.")
+            if not args.quiet:
+                print(f"'{asset}' not updated.")
 
     # commit changes
     staged = sorted(repo.files_staged)
     if staged:
-        print(repo._diff_changes())
-        if request_user_response("Save changes? No discards all changes. (y/n) "):
-            repo.commit('edit asset(s).', staged)
-            repo.commit(repo.generate_commit_message(message=args.message, cmd="edit"))
+        if not args.quiet:
+            print(repo._diff_changes())
+        if args.yes or request_user_response("Save changes? No discards all changes. (y/n) "):
+            repo.commit(repo.generate_commit_message(message=args.message,
```

Comment:
That is not true, the result will either just be `message`, if `-m` is used by the user, or it will be `cmd: staged_files keys -> destination`. Maybe there might be more fitting variable names to reflect this.